pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: nordic_dfu
 description: This library allows you to do a Device Firmware Update (DFU) of your nrf51 or nrf52 chip from Nordic Semiconductor. Fork of flutter-nordic-dfu.
-version: 5.2.1
+version: 5.3.0
```

Comment:
Its best to use version 6.0.0 for this change.

Reply:
Updated to 6.0.0